js/src/textures/TextTexture.js

```diff
@@ -0,0 +1,52 @@
+var _ = require('underscore');
+var Promise = require('bluebird');
+var TextTextureBase = require('./TextTexture.autogen');
+
+var TextTextureView = TextTextureBase.TextTextureView.extend({});
+
+var TextTextureModel = TextTextureBase.TextTextureModel.extend({
```

Comment:
Can you add this to the list in the README under the `# Overridden classes` section?